### DISCUSSION TITLE
Allow TextField to be focused

### DISCUSSION THREAD

**Discussion by AlexDicy** [wix/react-native-ui-lib#2394] [Ideas]
EDIT: I think the issue was caused by me wrapping the TextField component and not passing the right props.

Currently, TextField does not expose any way to be focused

**Comment by ethanshar** [comment]
TextField suppose to have the same focus functionality as RN TextInput component
You can either pass the `focus` prop to focus initially or passing a ref and focus the input manually.

Good luck!

**Reply by AlexDicy** [reply]
Thank you, I went ahead and tested it with both versions of TextField:
```tsx
const textFieldRef = React.useRef<TextInput>();
return <View padding-20 paddingV-80>
    <TextField ref={textFieldRef} marginT-16/>
    <Button marginT-26 text="Focus" onPress={() => textFieldRef.current?.focus()}/>
</View>;
```
This works fine, even when using `migrate`. `onSubmitEditing` also work as intended.

I should have provided an example code in the main post, because I'm not able to reproduce the issue anymore and it seems like it is working fine.
